src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -17,6 +17,50 @@
 
     z-index: $menu-z-index;
     margin-top: 5px;
-    perspective: 500px;
-    perspective-origin: center top;
+}
+
+@include breakpoint($medium-screen) {
+    .bdl-DropdownMenu--responsive {
+        &.dropdown-menu-element {
+            right: 0;
+            bottom: 0;
+            left: 0;
+            margin-top: 0;
+        }
+
+        &.dropdown-menu-enabled {
+            // cancels out the tether inline styling without having to set enabled=false
+            transform: none !important;
+        }
+
+        .aria-menu:not(.submenu) {
+            height: 100%;
+            margin: 0;
+            padding-top: $bdl-header-height + $bdl-grid-unit * 3;
+            background-color: $bdl-gray-02;
+            border: none;
+            border-radius: 0;
+            box-shadow: none;
+
+            .scroll-container {
+                height: calc(100vh - #{$bdl-header-height});
+                max-height: calc(100vh - #{$bdl-header-height});
```

Comment:
Do we need `max-height` here if it's being set to the same value as `height`?

Reply:
yeah, kinda pointless. I could have instead unset max-height